contracts/compile.sh

```diff
@@ -39,7 +39,9 @@ contracts="PaymentVault \
   IEigenDAServiceManager \
   MockRollup \
   EjectionManager \
-  EigenDACertVerifier \
+  EigenDACertVerifierV1 \
+  EigenDACertVerifierV2 \
+  EigenDACertVerifierV1_V2 \
```

Comment:
will the decoupling of V1 cert verification from V2 require existing rollups leveraging the V1 cert verifier we've already deployed to deploy a new one?

Reply:
Nah, they can just use whatever already works for them. If they want to transition, they can deploy a new one at their leisure. The new cert verifiers should remain functionally the same, as this PR strives NOT to change verification logic, exposed function selectors for verification, and behaviors.